app/javascript/lib/study-overview.js

```diff
@@ -179,36 +167,47 @@ function renderScatterPlot(target, rawPlot) {
       $(`#${plotId}`).data('camera', newCamera)
     }
   })
-
-  window.SCP.scatterPlotLayout = layout
 }
 
-/** Fetch and draw scatter plot for default Explore tab view */
-async function drawScatterPlot() {
-  const spinnerTarget = $('#plots')[0]
+/** draws scatter plot */
+async function drawScatterPlot(accession, cluster, plotIndex) {
+  const plotId = `cluster-plot-${plotIndex}`
+  const legendId = `cluster-legend-${plotIndex}`
```

Comment:
typically it's good form to not have parallel indexes like this, and if you need two things to be related, you either create an object for them or use subselectors within a shared parent container (e.g. the legend DOM element would be found by $(parentContainerId).find('.legend-container')

I don't think that's important to fix here though, since this code will be getting migrated to React

Reply:
Good point, I added a comment to this effect in 555d1f9.